tests/fixtures/functional/plugins/no-namespace/plugin-example/composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "extra": {
         "terminus": {
-            "compatible-version": "1.*"
+            "compatible-version": "2.*"
```

Comment:
If you want to maintain these tests on the 1.x branch, you could make this `^1|^2`. Not too important, though.